### ISSUE TITLE
Uncaught TypeError: Cannot read property 'disabled' of null

### ISSUE THREAD

**Issue by sammkj** [fians/Waves#139]
Error occurs when I click outside of the element that contains waves-effect. For example:

```
    <body>
      <div class="waves-effect>haha</div>
    </body>
```
When I click on ```body```, ```Uncaught TypeError: Cannot read property 'disabled' of null``` pops up. It seems like there's a problem in the ```showEffect``` method.

To reproduce:

1. $ clone https://github.com/komorebi-works/meteor-waves
2. $ cd ~/test
3. $ ln -s ~/meteor-waves packages/waves
4. $ meteor

**Comment by peruzzo** [comment]
+1

**Comment by fians** [comment]
Fixed by @xiaoyuze88 #140
